config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
```

Comment:
We usually utilize helm charts to deploy our apps. Is there a plan to add helm chart to this repo, or is this on GiantSwarm's side to do?